app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,18 @@
+class RubocopController  < ApplicationController
+	def initialize
+		print "this is so bad patern"
+	end
+
+	def print_fib(range)
+		range.each do |n|
+			p fib(n)
+		end
+	end
+
+	private
+		def fib(n)
+			return 1 if n <= 1
+			fib(n-2) + fib(n-1)
+      "hogehogeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee"
```

Comment:
__[rubocop]__
* Style/IndentationConsistency: Inconsistent indentation detected.
* Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)